x/valset-pref/msg_server.go

```diff
@@ -101,3 +101,26 @@ func (server msgServer) WithdrawDelegationRewards(goCtx context.Context, msg *ty
 
 	return &types.MsgWithdrawDelegationRewardsResponse{}, nil
 }
+
+func (server msgServer) DelegateBondedTokens(goCtx context.Context, msg *types.MsgDelegateBondedTokens) (*types.MsgDelegateBondedTokensResponse, error) {
```

Comment:
Would it be possible to add godoc for this?

Reply:
Will do. I have followed this pattern of short godoc in msg_server function and more descriptive doc in it's helper keeper method. lmk if that's a good approach